src/api/java/com/minecolonies/api/colony/buildings/modules/settings/ICraftingSetting.java

```diff
@@ -0,0 +1,47 @@
+package com.minecolonies.api.colony.buildings.modules.settings;
+
+import com.minecolonies.api.colony.buildings.IBuilding;
+import com.minecolonies.api.colony.buildings.views.IBuildingView;
+import com.minecolonies.api.colony.requestsystem.token.IToken;
+import com.minecolonies.api.crafting.IRecipeStorage;
+import net.minecraft.item.ItemStack;
+
+import java.util.List;
+
+/**
+ * String Setting.
+ */
+public interface ICraftingSetting extends ISetting
+{
+    /**
+     * Get the setting value.
+     * @return the current value.
+     */
+    IToken<?> getValue(final IBuilding building);
+
+    /**
+     * Get the setting value.
+     * @return the current value.
+     */
+    IRecipeStorage getValue(final IBuildingView building);
```

Comment:
Seems a bit weird this has different return types between client/server interfaces...  and inconsistent with `set`

Reply:
I agree, let me standardize that